### ISSUE TITLE
[Sticky] Feature Request: Cooldown on reposting sticky message

### ISSUE THREAD

**Issue by Q-out** [Tobotimus/Tobo-Cogs#78]
Please add a config for cooldown/delay for Sticky cog so that the cog posts the sticky only after a set amount of time has passed since the last time it posted it. 

Currently it deletes and reposts immediately after each message which makes even small conversations very annoying to hold in the channel. I'm aware this request is like a duplicate of or very similar to #74 but a simple cooldown timer might be strictly easier to implement and will make sticky have utility in more use cases.

**Comment by Tobotimus** [comment]
Hi @Q-out, thanks for opening this feature request.

I've added a 3-second cooldown to Sticky, so messages should never be posted in a channel more often than 3 seconds at a time.

You're right that this is a much simpler approach than #74, in terms of both implementation and user experience. So I'm sticking with this one over #74 :)
